modules/virt-about-services.adoc

```diff
@@ -13,7 +13,11 @@ ClusterIP:: Exposes the service on an internal IP address within the cluster. `C
 NodePort:: Exposes the service on the same port of each selected node in the cluster. `NodePort` makes a service accessible from outside the cluster.
 
 LoadBalancer:: Creates an external load balancer in the current cloud (if supported) and assigns a fixed, external IP address to the service.
-
++
+[NOTE]
+====
+For on-premise clusters, you can configure a load-balancing service by using the MetalLB Operator in layer 2 mode. The BGP mode is not supported. The MetalLB Operator is installed in the `metallb-system` namespace. See the _Additional resources_ section for more information.
```

Comment:
s/load-balancing/load balancing (according to the [RH supplementary style guide](https://redhat-documentation.github.io/supplementary-style-guide/))